README.md

```diff
@@ -1 +1,70 @@
-# Too Many Hooks
+# <img src="./assets/too-many-hooks-outlined.svg" alt="Too Many Hooks" width="50%">
+
+An opinionated collection of typed React hooks
+
+## Storybook
+
+- The Storybook for the current main version can be found at: [https://dawsonbooth.github.io/too-many-hooks](https://dawsonbooth.github.io/too-many-hooks)
+- Storybooks will also be build and deployed for each PR at `https://dawsonbooth.github.io/too-many-hooks/pr-preview/pr-[PR number]`
+
+## Guiding Principles
+
+- There should be one obvious way to do each thing
+- Hook functionality can overlap as long as they each have a unique defined purpose
+- Features should be self-contained
```

Comment:
I think for the sake of clarity we should either delete "Features should be self-contained" or reword it so that it's clear when to make room for composing other hooks we've defined if we need (e.g. `useArray` within `useFlexCorners`) vs when to re-define functionality as in `useArray` and `useQueue`.

Reply:
> - Hook functionality can overlap as long as they each have a unique defined purpose

To that point, and perhaps for the pithy conciseness I mentioned in another comment, maybe we should remove this and either document the effort elsewhere or let that be self-evident in our code.